ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/log/state/AtlasDumpBegin.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.hadoop.hive.ql.parse.repl.dump.log.state;
 
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
```

Comment:
How about grouping with the line 25 `import com.fasterxml.jackson.annotation.JsonProperty;`? A same comment applies for other files, too.